srcpkgs/sway/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'sway'
 pkgname=sway
-version=1.0
+version=1.1.1
 revision=1
 build_style=meson
 conf_files="/etc/sway/config"
 hostmakedepends="pkg-config wayland-devel scdoc git"
 makedepends="wlroots-devel wayland-devel wayland-protocols pcre-devel
  json-c-devel pango-devel cairo-devel gdk-pixbuf-devel"
-depends="xorg-server-xwayland"
+depends="swaybg xorg-server-xwayland"
```

Comment:
Technically it functions without swaybg.

Reply:
Note we do not have swayidle or swaylock as deps. Could be useful create a sway meta package.

Reply:
From @ddevault in 1.1 desc:

> swaybg is a required dependency of sway. Packagers are encouraged to configure it as such when updating sway for their distribution.

Reply:
Ah! Thanks for the clarification.